### PR TITLE
Update extension.ts

### PR DESCRIPTION
**Pull request by Fndroid**
Fix package sync fail in Windows.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import * as vscode from 'vscode';
+import * as p from 'path';
```
Comment (cyanzhong) [inline src/extension.ts]: `p` looks too simple to me, it's not easy to understand 😂

Comment (Fndroid) [reply src/extension.ts]: check the latest commit

Comment (cyanzhong) [reply src/extension.ts]: awesome, really appreciate it.

**Review by cyanzhong — APPROVED** [review]
LGTM 👍

**Pull request merged**